Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/CSGSceneBottomGUI.cs

```diff
@@ -13,670 +13,98 @@ namespace Chisel.Editors
-		}
-		#endregion
-
-		#region ScaleSnapping
-		static void ScaleSnappingGUI(ref Rect position, bool largeLabels, bool showUnits, bool showIncDec)
-		{
-			Space(ref position, 2);
-
-			//
-			// Scale snapping
-			//
-			CSGEditorSettings.ScaleSnapping = Toggle(ref position, CSGEditorSettings.ScaleSnapping, largeLabels ? scaleSnappingContent : scaleContent);
-			if (CSGEditorSettings.ScaleSnapping)
-			{
-				CSGEditorSettings.ScaleSnap = FloatField(ref position, CSGEditorSettings.ScaleSnap, kFloatFieldWidth);
-				if (showIncDec)
-				{
-					if (TinyButton(ref position, increaseContent, EditorStyles.miniButtonMid)) IncreaseScaleSnap();
-					if (TinyButton(ref position, decreaseContent, EditorStyles.miniButtonRight)) DecreaseScaleSnap();
-				}
-				if (showUnits) Label(ref position, Units.PercentageUnitContent);
-			}
-		}
-
-		static float ScaleSnappingWidth(bool largeLabels, bool showUnits, bool showIncDec)
-		{
-			return 2 +
-					StyleWidth(EditorStyles.toolbarButton, largeLabels ? scaleSnappingContent : scaleContent) +
-					StyleWidth(EditorStyles.toolbarTextField, minWidth: kFloatFieldWidth) +
-					(showIncDec ? (StyleWidth(EditorStyles.miniButtonMid, increaseContent) +
-								   StyleWidth(EditorStyles.miniButtonRight, decreaseContent)) : 0) +
-					(showUnits ? StyleWidth(miniLabel, Units.PercentageUnitContent) : 0);
-		}
-		#endregion
-
-		#region AxisLocking
-		static void AxisLockingGUI(ref Rect position, bool showLockLabel)
-		{
-			Space(ref position, kButtonSpace);
-
-			//
-			// Axis locking
-			//
-			if (showLockLabel) Label(ref position, lockContent);
-			CSGEditorSettings.AxisLockX = Toggle(ref position, CSGEditorSettings.AxisLockX, lockXAxisContent, lockButtonToggleColor, lockXButtonStyle);
-			CSGEditorSettings.AxisLockY = Toggle(ref position, CSGEditorSettings.AxisLockY, lockYAxisContent, lockButtonToggleColor, lockYButtonStyle);
-			CSGEditorSettings.AxisLockZ = Toggle(ref position, CSGEditorSettings.AxisLockZ, lockZAxisContent, lockButtonToggleColor, lockZButtonStyle);
-		}
-
-		static float AxisLockingWidth(bool showLockLabel)
-		{
-			return kButtonSpace +
-					(showLockLabel ? StyleWidth(miniLabel, lockContent) : 0) +
-					StyleWidth(lockXButtonStyle, lockXAxisContent) +
-					StyleWidth(lockYButtonStyle, lockYAxisContent) +
-					StyleWidth(lockZButtonStyle, lockZAxisContent);
-		}
-		#endregion
-
-		#region ViewOptions
-		static void ViewOptionsGUI(ref Rect position)
-		{
-			Space(ref position, kButtonSpace);
-
-			//
-			// View Options
-			//
-			MaskField(ref position, viewContent, CSGEditorSettings.ViewOptions, ChangeViewOptions, false);
-		}
-
-		static float ViewOptionsWidth()
-		{
-			return kButtonSpace +
-					StyleWidth(EditorStyles.toolbarDropDown, viewContent);
-		}
-		#endregion
-
-		static void DrawLeftButtons(Rect position, SceneViewBarState state)
-		{
-			directionForward = true;
-
-			position.x--;
-
-			ViewOptionsGUI(ref position);
```

Comment:
The view options, I can understand why you left them out at the moment since they're not fully implemented, but they allowed for all kinds of debug views that would be nice to put back at some point

Reply:
Yeah, as soon as there's more to change we can add a menu back.